### DISCUSSION TITLE
Warn Auto-Rebase is enabled during PR approval

### DISCUSSION THREAD

**Discussion by LiandriCorp** [community/community#53155] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Our team often approves a PR with comments with the trust that those simple comments will be addressed before merging. This greatly speeds up dev time by reducing back-and-forth. But one can easily approve a PR in this way, not knowing that auto-merge/rebase was enabled.

If auto-merge/rebase is enabled, we'd love a simple little message or warning prior to accepting approval so we can take appropriate action.

**Comment by premiumconstructionpro** [comment]
You can apply, allow, or disable the commit base change for all pull request merges on GitHub.com in your repository[.](https://premiumconstructionpro.com/) You can configure pull request merging options on GitHub.com to suit your needs and workflow preferences for managing Git history.
